src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUNDARY_RANDOM = 100;
+
+    public Ball getRandomBall() {
+        int generatedNumber = new Random().nextInt(BOUNDARY_RANDOM);
```

Comment:
create Random and ColorSupplier at class level

Reply:
@okuzan Sorry, could you explain why you need to create instances of Random and ClassSupplier at the class level?